lua/autorun/webaudio.lua

```diff
@@ -507,6 +535,23 @@ local function isWhitelistedURL(url)
 	return false
 end
 
+local function isWhitelistedURL(url)
+	if not isstring(url) then return false end
+
+	local relative = url:match("^https?://(.*)")
+	if not relative then return false end
+
+	if CLIENT and CustomWhitelist then
+		local isWL = checkWhitelist(LocalWhitelist, relative)
+		if not isWhite then
+			warn("%s failed because it is not in the local Whitelist!",  url)
```

Comment:
Shouldn't be warning the user when the server is just checking if the URL is whitelisted, we already warn in https://github.com/Vurv78/WebAudio/blob/6977c0894bd11d8e8491aaa87e624d73c2aee3ad/lua/webaudio/receiver.lua#L76

Reply:
I wanted a warning so you knew what whitelist caused error.